docker-compose.override.yml.example

```diff
@@ -16,12 +16,12 @@ services:
   app:
     ports:
       - "8080:8000"		# allows to run app on different port on host
-    user: 1003:1003		# set current user id so it has access to mounted volumes
+    user: 1001:1001		# set current user id so it has access to mounted volumes
 
   dev:
     ports:
       - "8080:8000"
-    user: 1003:1003
+    user: 1001:1001
```

Comment:
Dit is de example, deze change voegt inhoudelijk weinig toe

Reply:
Klopt, maar de dingen worden in de container met de user 1001 gedaan, dus deze example werkt zowaar.. :+1: